xrpl/models/transactions/check_create.py

```diff
@@ -19,21 +12,36 @@
 @dataclass(frozen=True)
 class CheckCreate(Transaction):
     """
-    Represents a CheckCreate transaction on the XRP ledger, which
-    creates a Check object. A Check object is a deferred payment
+    Represents a `CheckCreate <https://xrpl.org/checkcreate.html>`_ transaction,
+    which creates a Check object. A Check object is a deferred payment
     that can be cashed by its intended destination. The sender of this
     transaction is the sender of the Check.
-
-    `See CheckCreate <https://xrpl.org/checkcreate.html>`_
     """
 
-    #: This field is required.
+    #: The address of the `account
+    #: <https://xrpl.org/accounts.html>`_ that can cash the Check. This field is
```

Comment:
The weird line break here is to work around a quirk of how Sphinx processes these lines.

If it were something like this, it would glitch out:

```
    #: The address of the `account <https://xrpl.org/accounts.html>`_ that
    #: can cash the Check. This field is
```

The build returns an error message like this if you do that:

> docstring of xrpl.models.transactions.CheckCreate.destination:4: WARNING: Inline interpreted text or phrase reference start-string without end-string.

I'm not sure why, breaking the line in the middle of the link works around this problem.